### PR TITLE
Frontend/animating background

### PR DESCRIPTION
**Pull request by karolszymonczyk**
Look improvements

### CODE REVIEW

**Review by pawelrubin — APPROVED** [review]

File: frontend/package.json [hunk 1]
```diff
@@ -47,6 +48,7 @@
     ]
   },
   "devDependencies": {
+    "@fortawesome/fontawesome-free": "^5.11.2",
```
Comment (wadamek65) [inline frontend/package.json]: I don't think this should be in `devDependencies`

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,29 +1,84 @@
...
-              <Redirect to="register" />
-            </Route>
-          </Switch>
-        </>
-      </BrowserRouter>
+      <ParticlesComponent />
+      <div className="foreground">
+        <MuiThemeProvider theme={theme}>
+          <BrowserRouter>
+            <>
```
Comment (wadamek65) [inline frontend/src/App.js]: Fragment here is unnecessary I think

File: frontend/src/components/Auth/Register/Register.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ const Register = () => {
   return (
     <div className="Auth">
       <div className="card">
-        <h1>Sudoku Battle Royale</h1>
+        <h1 style={{ color: "white" }}>Sudoku Battle Royale</h1>
```
Comment (wadamek65) [inline frontend/src/components/Auth/Register/Register.js:36]: It is better to keep one convention of styling. Mixing inline-styles and css classes will quickly become very hard to maintain. I would suggest exporting this style to a css file and adding a class here (even if it is only this one property).

File: frontend/src/components/ParticlesComponent/ParticlesComponent.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+canvas {
+  height: 100vh !important;
```
Comment (wadamek65) [inline frontend/src/components/ParticlesComponent/ParticlesComponent.scss]: Usually `!important` is a red flag. This should be avoided at all costs. Why did you add this here?

**Review by pawelrubin — APPROVED** [review]

File: frontend/src/index.css [hunk 5]
```diff
@@ -5,9 +5,27 @@ body {
...
+  -webkit-box-shadow: 0 0 0 30px rgba(0, 0, 0, 0.8) inset !important;
+}
+
+/*Change text in autofill textbox*/
+input:-webkit-autofill {
+  -webkit-text-fill-color: white !important;
+}
+
+/*Removes blue border on buttons click*/
+button:focus {outline: none !important;}
```
Comment (wadamek65) [inline frontend/src/index.css]: How about adding `button { all: unset }` to get rid of all native button styles and then adding them yourself? `!important` should be used with caution.

**Review by wadamek65 — APPROVED** [review]

File: frontend/src/components/Auth/Register/Register.js (hunk 3 above)
Comment (karolszymonczyk) [reply frontend/src/components/Auth/Register/Register.js:36]: it's only temporary. In this place will be logo image.

File: frontend/src/App.js (hunk 2 above)
Comment (karolszymonczyk) [reply frontend/src/App.js]: You told me to crate Fragment there. But ok I'll remove it.

File: frontend/src/index.css (hunk 5 above)
Comment (karolszymonczyk) [reply frontend/src/index.css]: It also does not work without !important

Comment (karolszymonczyk) [reply frontend/src/index.css]: But i found another way without !important
